packages/multichain-network-controller/package.json

```diff
@@ -48,15 +48,19 @@
   },
   "dependencies": {
     "@metamask/base-controller": "^8.0.0",
+    "@metamask/controller-utils": "^11.7.0",
     "@metamask/keyring-api": "^17.4.0",
+    "@metamask/keyring-utils": "^3.0.0",
     "@metamask/utils": "^11.2.0",
-    "@solana/addresses": "^2.0.0"
+    "@solana/addresses": "^2.0.0",
+    "loglevel": "^1.8.1"
   },
   "devDependencies": {
     "@metamask/accounts-controller": "^27.0.0",
     "@metamask/auto-changelog": "^3.4.4",
     "@metamask/keyring-controller": "^21.0.1",
     "@metamask/network-controller": "^23.2.0",
+    "@metamask/superstruct": "^3.1.0",
```

Comment:
Does this need to be in `dependencies` instead of `devDependencies`?

Reply:
I'd say yes, given that we are using `is`/`assert` functions from this package

Reply:
Re-opening this thread, cause we still to move this one into `dependencies`.